app/repository/hbase/legalunit/LegalUnitColumns.scala

```diff
@@ -0,0 +1,9 @@
+package repository.hbase.legalunit
+
+object LegalUnitColumns {
+  val uBRN = "UBRN"
```

Comment:
maybe have uBRN as ubrn instead may look cleaner

Reply:
Agree, uBRN looks ugly, change to ubrn throughout the codebase